README.md

```diff
@@ -1,41 +1,62 @@
-# Microsoft Container Registry (MCR)
+# Microsoft Artifact Registry (MAR)
 
-Microsoft Container Registry (MCR) is the primary Registry for all Microsoft Published docker images that offers a reliable and trustworthy delivery of container images with a syndicated catalog, while maintaining the quality that customers expect from a  Microsoft product offering. 
+Microsoft Artifact Registry (also known as Microsoft Container Registry or MCR) is the primary Registry for all Microsoft Published artifacts (such as container images) that offers a reliable and trustworthy delivery of artifacts with a syndicated catalog, while maintaining the quality that customers expect from a  Microsoft product offering. 
 
-For more background on MCR:
+For more background on MAR:
 
 - [Microsoft syndicates container catalog (mcr.microsoft.com)](https://azure.microsoft.com/blog/microsoft-syndicates-container-catalog/)
 - [Improved discovery experience for Microsoft containers on Docker Hub
 ](https://cloudblogs.microsoft.com/opensource/2019/01/17/improved-discovery-experience-microsoft-containers-docker-hub/)
 
-## Browsing MCR Content
+## Browsing MAR Content

```

Comment:
Should we have a separate section that maintains the information about programmatically querying the data using the tags and catalog endpoints?

Reply:
From the previous comments, from my understanding, we want users to query results from the portal

Reply:
Ah, missed those ones